src/utils.c

```diff
@@ -48,6 +48,7 @@
 #include <stdio.h>                      // Required for: FILE, fopen(), fseek(), ftell(), fread(), fwrite(), fprintf(), vprintf(), fclose()
 #include <stdarg.h>                     // Required for: va_list, va_start(), va_end()
 #include <string.h>                     // Required for: strcpy(), strcat()
+#include <ctype.h>                      // Required for: isalnum()
```

Comment:
In raylib I try to minimize standard library dependency, I prefer to avoid it.

Reply:
Alright, I'll find a workaround, thanks

Reply:
I can review it if you want.